_layouts/team-member.html

```diff
@@ -42,12 +42,47 @@ <h1 class="title">{{ page.firstname }} {{ page.lastname }}
                             {{ link.name }}
                         </a>
                     </li>
-                    {% endfor %}
+                    {% endfor %}    
                 </ul>
             </div>
             {% endif %}
         </div>
     </div>
+
+    <!-- blog posts -->
+    <h3 class="title"> Blog posts that {{ page.firstname }} has written: </h3>
```

Comment:
Hm, so I think this is the inverse of the change that we wanted. Keep in mind that headings and subheadings nest; so if you want a heading to be "within" the blog posts that a user has written, the number should increment by one.

In other words - this should be an `<h2>` tag; the heading inside `<div class="post-info">` should be a `h3`!